foundry/foundry.py

```diff
@@ -854,9 +849,54 @@ def toTorch(self, raw=None, split=None):
                 for key in df.keys():
                     arr.append(df[key].values)
 
-            return FoundryDataset_Torch(inputs, targets)
+            return TorchDataset(inputs, targets)
         else:
             raise NotImplementedError
+
+    def to_tensorflow(self, split: str = None):
+        """Convert Foundry Dataset to a Tensorflow Sequence
+
+        Arguments:
+            split (string): Split to create Tensorflow Sequence on.
+                    **Default:** ``None``
+
+        Returns: (TensorflowSequence) Tensorflow Sequence of all the data from the specified split
+
+        """
+        from foundry.loaders.tf_wrapper import TensorflowSequence
+        
+        raw = self.load_data(as_hdf5=False)
+        
+        if not split:
+            split = self.dataset.splits[0].type
+
+        if self.dataset.data_type.value == "hdf5":
+            inputs = []
+            targets = []
+            for key in self.dataset.keys:
+                if len(raw[split][key.type][key.key[0]].keys()) != self.dataset.n_items:
+                    continue
+
+                val = np.array([raw[split][key.type][key.key[0]][k] for k in raw[split][key.type][key.key[0]].keys()])
+                if key.type == 'input':
+                    inputs.append(val)
+                else:
+                    targets.append(val)
+
+            return TensorflowSequence(inputs, targets)
+        elif self.dataset.data_type.value == "tabular":
```

Comment:
Another quick thought: What do you think of moving this code, which is specific to the TFSequence to a @classmethod of TensorflowSequence rather than inside of this function?

Reply:
I think this is done?